### PR TITLE
ResizeBicubic added

### PR DESCRIPTION
**Pull request by alexanderst**
1. ResizeBicubic wrapper added
2. Some fixes in nd4j ops wrappers
3. Fixed import tests removed from ignore section
4. Some fixes and cleanup in tests

### CODE REVIEW

File: nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/FakeQuantWithMinMaxVarsPerChannel.java [hunk 1]
```diff
@@ -21,30 +21,60 @@
...
+        addBArgument(narrow);
+    }
+
+    public FakeQuantWithMinMaxVarsPerChannel(INDArray x, INDArray min, INDArray max) {
+        Preconditions.checkArgument(min.isVector() && max.isVector() && min.length() == max.length(),
+                "FakeQuantWithMinMaxVarsPerChannel: min and max should be 1D tensors with the same length");
+        addInputArgument(x,min,max);
+        addIArgument(8);
+        addBArgument(false);
+    }
```
Comment (AlexDBlack) [inline nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/FakeQuantWithMinMaxVarsPerChannel.java:57]: These 3 constructors could be cleaned up into:
```
public FakeQuantWithMinMaxVarsPerChannel(INDArray x, INDArray min, INDArray max, int num_bits){
    this(x, min, max, num_bits, false);
}

public FakeQuantWithMinMaxVarsPerChannel(INDArray x, INDArray min, INDArray max, boolean narrow) {
    this(x, min, max, 8, narrow);
}

public FakeQuantWithMinMaxVarsPerChannel(INDArray x, INDArray min, INDArray max, int num_bits, boolean narrow) {
    ...
}
```

**Review by AlexDBlack — CHANGES_REQUESTED** [review]
Looks good other than one tiny cleanup :+1:

File: nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/FakeQuantWithMinMaxVarsPerChannel.java (hunk 1 above)
Comment (alexanderst) [reply nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/FakeQuantWithMinMaxVarsPerChannel.java:57]: Fixed.

**Review by AlexDBlack — APPROVED** [review]
:+1:

**Pull request merged**